website/src/components/Features.tsx

```diff
@@ -6,29 +6,36 @@ import Image from 'next/image';
 import sampleImage from '@/images/sampleImage.png';
 import { Button } from '@/components/Button';
 import { handleSubscription } from '@/services/subscribeService'; // Import subscription service
-import { handleSendWelcomeEmail } from '@/services/emailService'; // Import email service
 
 export function Features() {
   const { t } = useTranslation();
   const [email, setEmail] = useState('');
   const [message, setMessage] = useState('');
-  const [isSubmitting, setIsSubmitting] = useState(false);
 
-  const handleEmailSubmission = async (e: React.FormEvent) => {
+  const HandleEmailSubmission = async (e: React.FormEvent) => {
```

Comment:
Please use lower camel case for function naming

```suggestion
  const handleEmailSubmission = async (e: React.FormEvent) => {
```